### PR TITLE
docs: updated the description of min_quorum

### PR DESCRIPTION
**Pull request by neenap**
Added in clarification on how autopilot uses min_quorum

### CODE REVIEW

**Review by karl-cardenas-coding — APPROVED** [review]
Looks good 👍🏼

File: website/content/docs/agent/options.mdx [hunk 1]
```diff
@@ -919,7 +919,7 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
     unhealthy. Defaults to 250.
 
   - `min_quorum` - Sets the minimum number of servers necessary
-    in a cluster before autopilot can prune dead servers. There is no default.
+    in a cluster. Autopilot will stop pruning dead servers when this minimum is reached. There is no default.
```
Comment (FFMMM) [inline website/content/docs/agent/options.mdx:922]: This LGTM; 

As I was parsing thru @banks's comment [here](https://github.com/hashicorp/consul/issues/8118#issuecomment-645330040), I was thinking if we could encourage practitioners to set a reasonable default, given N := number of servers

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/550147.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 209d03c26771745696244ca07846264494a32dd6 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 209d03c26771745696244ca07846264494a32dd6 onto `release/1.11.x` succeeded!

**Pull request merged**